andela/ah-92explorers-backend
# #166790010 Users Should Be Able to See Time It Takes To Read An Article

#### What does this PR do?
- Adds article read time in human readable string

#### Description of Task to be completed
- Enables users to view article read time

#### How should this be manually tested?
```shell
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-article-read-time-166790010
npm install
npm start
```
- Navigate to `localhost:3000/api/articles/:slug`
- check the for `time` field in the response body

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[166790010](https://www.pivotaltracker.com/story/show/166790010)
#### Screenshots (if appropriate)
![Imgur](https://i.imgur.com/QWnAneU.png)
#### Checklist
